data/dimensionless_mappings.yaml

```diff
@@ -0,0 +1,5 @@
+# In general:
+# model_variable_name:
+#   cmor_unit_string: pint_friendly_SI_units
+sst:
```

Comment:
Sorry guys, the issue I posted has the wrong example, sst does not exist as a var in the tables right? We can switch this to ´so ´ for example (see table from @siligam)

Reply:
sst would be tos

Reply:
thanks @christian-stepanek! then let's use `tos`

Reply:
```suggestion
tos:
```

Reply:
Sorry, I just wanted to solve the mystery with sst, not cause more confusion. Tos is not really a good example as it would not cause any trouble since it would either not have to be converted at all or to/from °C/K. Variables so (salinity of the ocean) and sos (salinity of the ocean surface) would be better candidates, right?

Reply:
correct!